libs/tests/helpers.js

```diff
@@ -42,17 +42,19 @@ const audiusLibsConfig = {
   isServer: true
 }
 
-async function initializeLibConfig (ownerWallet) {
+async function initializeLibConfig (ethOwnerWallet, dataOwnerWallet) {
```

Comment:
this is also called in stakingTest - i think this signature change would break that call right

Reply:
unsure but good q

Reply:
actually no - this would not break anything as dataOwnerWallet from stakingTest would be null, resulting in equivalent behavior. only the userReplicaSetManagerTest.js actually passes a value in here